exhaustive.cabal

```diff
@@ -16,24 +16,31 @@ build-type:          Simple
 extra-source-files:  README.md
```

Comment:
If you like, but I think this is mostly your work at this point, give or take a helper function or two :smile: